solutions/CSES Maximum Subarray II.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: cses-1671
+title: CSES - Maximum Subarray II
+author: Neo Wang
+---
+
+# Problem 
+
+We are asked to find the maximum subarray with size in $[a,b]$. 
+
+<Spoiler title="Hint 1">
+
+We can observe that we will be performing sum range queries for our subarray.
+
+Therefore, we should construct a prefix sum array to perform these queries.
+
+</Spoiler>
+
+# Solution
```

Comment:
if the hint is wrapped in a spoiler tag, then shouldn't the solution be hidden by a spoiler too?